l10n_it_rea/README.rst

```diff
@@ -33,6 +33,7 @@ Contributors
 * Alessio Gerace <alessio.gerace@agilebg.com>
 * Nicola Malcontenti <nicola.malcontenti@agilebg.com>
 * Lorenzo Battistini <lorenzo.battistini@agilebg.com>
+* Andrea Gallina <a.gallina@apuliasoftware.it>
```

Comment:
Could you please take the occasion to apply to the README file the following changes?

- add `:target: http://www.gnu.org/licenses/agpl-3.0-standalone.html `to the badge license 
- put ============ above REA Register
- remove content of Usage section to the previous section
- add Try me on Runbot lunk to Usage section
-  use https://odoo-community.org

See https://raw.githubusercontent.com/andrea4ever/l10n-italy/5be3170070d5c8b54b945b51f673dcd8d7b2c468/l10n_it_rea/README.rst as refernce